### DISCUSSION TITLE
Warum ist das Systemprotokoll leer?

### DISCUSSION THREAD

**Discussion by idominiki** [OpenCCU/OpenCCU#2600] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe das Problem, dass das Systemprotokoll dauerhaft leer ist. Leider treten jedoch immer mal wieder Störungen auf, die doch aber dann dort angezeigt werden sollten, oder?

Hat einer einen Tipp woran dies liegen könnte?

Viele Grüße
Dominik
![Bildschirmfoto 2024-01-10 um 17 17 05](https://github.com/jens-maus/RaspberryMatic/assets/62521486/e840ec43-31f0-4ef0-bd47-5e7983c67039)
![Bildschirmfoto 2024-01-10 um 17 15 33](https://github.com/jens-maus/RaspberryMatic/assets/62521486/bb53d86f-9eed-4c84-8517-8d343b014cec)

**Comment by jp112sdl** [comment, marked as answer]
Hi,
dein erster Screenshot zeigt Einstellungen vom "Fehlerprotokoll".
Der zweite dein leeres "Systemprotokoll".
Beide haben miteinander nichts zu tun.
Im Systemprotokoll tauchen nur Informationen von Objekten auf, bei denen du "Protokolliert" aktiviert hast (Systemvariablen, Geräte, Kanäle)
